### PR TITLE
UV excess in low mass stars

### PR DESCRIPTION
**Pull request by cpsanchez**
New branch for "UV excess in low mass stars"

### CODE REVIEW

**Comment by kelle** [comment]
This looks like a good start but I'm not sure I'm following the story. The learning goals are about UV excess but than the content is about light curves.  Let us know if you need help or when it's ready for review.

**Comment by cpsanchez** [comment]
Indeed I still want some guidance on how exactly we want to characterize and study the UV excess, Scott has told me that David Rodriguez may be the person to ask about this, but he is on holidays until the 14th, so for now I am looking online for different articles that may be relevant and completing the parts about accessing GALEX using gPhoton!

**Comment by dr-rodriguez** [comment]
I think this looks great. One thing I would perhaps change is that the mention that GALEX can be found at "the author's website" is very odd. GALEX is not a paper with an author and that's not even the mission website. I would rephrase that as `The GALEX catalog can be accessed via multiple places including this website, through MAST Casjobs, ...`

Also, we should probably list where we got the TWA stars from. I can't remember if I sent those to you or you grabbed them from a paper or from Simbad or elsewhere. Just for reproduceability in terms of the input data.

@ttdu you may want to have a look at this in general, too.

**Comment by cpsanchez** [comment]
Fixed it! Thank you for the comments @dr-rodriguez

**Comment by ttdu** [comment]
Hey @cpsanchez! I have some feedback for you as well; I tried reviewNB, but I had multiple comments per cell, so I'll Slack you a notebook that has my suggestions edited in.

Otherwise, two bigger things to mention:
- The GALEX Database is different from GUVcat. The Database is a huge collection of all the sources GALEX has ever observed, whereas GUVcat is a cleaned version of the Database created by Bianchi et. al that excludes multiple data entries per source (and has additional science info). I think this distinction gets lost a little bit in your first two sections -- I would always say GUVcat when you mean GUVcat, rather than 'The GALEX Catalog'
- In the notebook I'm Slacking to you, I've added a table of contents. I'd recommend using that as a starting place to re-organize some of the content, like adding some "sub-chapters" to improve readability. You'll see what I mean in the "simplified example" table of contents. When possible, you should also make the names focused on the **action** in that section (i.e. 'Getting Magnitudes of Nearby M-Dwarfs' instead of 'Nearby M-Dwarfs')

The notebook is looking great so far, and I hope my comments help you polish things up!

File: UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb [hunk 1]
```diff
@@ -0,0 +1,948 @@
+{
```
Comment (jrka) [inline UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb:1]: <p>I'd recommend splitting these up into multiple cells.</p><p> </p><p>For the learning goals, can you add some more specificity? For example, for "Download data" I presume (after having read the intro) this will specifically be GALEX spectra and we'll be using astroquery to download it from the MAST archive? [Haha, nope! I was wrong. I came back to this comment after reading through the rest of the tutorial.] </p><p> </p><p>It seems the learning goals might be more like....</p><ul><li>Load a catalog of star coordinates from a text file</li><li>Load a catalog of star coordinates from Vizier</li><li>Search GUVcat and Simbad for photometry of target stars given coordinates</li><li>Create a color-color diagram of stellar populations and interpret the results</li></ul><p> </p><p>That might not be extensive, there is probably more here that I am missing because the tutorial is very comprehensive, but those are the ones that stood out to me upon first review. </p><p> </p><p>As the reader I was also hoping for a bit more at the end of the introduction. I'm left wondering, of all thse cool things we can do with GALEX(look for variable, detect binaries, detect chromospheres), are we actually going to do one in the tutorial? Which one?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (jrka) [inline UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb:1]: <p>I am not at all familiar with pyvo! So this might also be a good thing to add to the introduction, give the reader the setup of 1) what they will be do (the science goals) and 2) the tools they will use for it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (jrka) [inline UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb:1]: <p>Recommend to change the title to say what we are going to do with GUVcat.</p><p> </p><p>Also need some information, either here or back up at the intro, about what GUVcat is and how we will use it.</p><p> </p><p>I appreciate the explanation of the naming conventon! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (jrka) [inline UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb:1]: <p>Why TWA 1? I realized through context this must be a star. So in the introduction, you could say that we plan to study TWA 1 and explain why you chose that one. Something interesting about it? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (jrka) [inline UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb:1]: <p>Can you add a markdown cell above this code block that explains what you are doing? I can follow along that you are specifying the coordinates you are looking for and you want results in a radius of 10 arcseconds. But, for example, will the user always use that URL to search GALEX though VO? Can you link to documentation to where a new user would have found that URL in the first place?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (jrka) [inline UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb:1]: <p>Oh, I had asked about TWA above, now I see it spelled out - be sure it's spelled out when the acronym is first used (above).</p><p> </p><p>I would make sure that either here, or ideally in the introduction above, a color-color diagram is defined and also its usefulness explained.</p><p> </p><blockquote>will be then to load this</blockquote><p> </p><p>I suggest remove "then".</p><p> </p><p>This section is really clearly explained with lots of detail! As the user I'm like, "Yes, I am following along!"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (jrka) [inline UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb:1]: <p>In retrospect, should loading the catalog of TWA star locations come first, and then search VO? Because that would be the scientific workflow... I've got a big list of stars, I want to know which ones have GALEX data. You could still demonstrate how to search VO with just the one star at first, and then do the loop for the whole list of stars.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (jrka) [inline UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb:1]: <p>Which bands do we have so far? FUV and NUV?</p><p> </p><p>I think we need to add what Simbad is, this is not raw data or observations we are querying, but published magnitudes. (I guess the same should be said about for GUVCat.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Comment (jrka) [inline UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb:1]: <blockquote>below the name 'FLUX_K' it indicates it is a magnitude!</blockquote><p>below the name 'FLUX_K' the unit indicates it is a magnitude!</p><p> </p><p>Rather than "problematic" values, are they "missing" values? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (jrka) [inline UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb:1]: <p>; instead of , before "we will"</p><p> </p><p>A bit more explanation will help here, since there are 20 lines of code coming up. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Comment (jrka) [inline UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb:1]: <p>Oops the output below says 41? Not sure if a typo or things changed! I found it helpful to not list specific numbers or outputs in my markdown cells in case they change in the future. So you can get around this by saying e.g. "We can check the length of the maK_TWA list to confirm that we have removed the missing values:"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Comment (jrka) [inline UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb:1]: <p>This is a good example of something that could be explained in the introduction. So we want to make a color-color diagram of two different populations: 1) the TW Hydra Asssociation, young stars whose coordinates are provided in a text file, and 2) a population of nearby M-dwarfs, old stars, the coordinates of which we'll get from a published catalog in Vizier. For both populations, we'll get the FUV, NUV magnitudes from GUVCat and the K, V magnitudes from Simbad.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Comment (jrka) [inline UV_excess_in_low_mass_stars/UV_excess_in_low_mass_stars.ipynb:1]: <p>This is so interesting! This is also information that could be in the introduction. What am I looking for when I look at a color-color diagram?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

**Comment by jrka** [comment]
<p>I made a number of in-line comments in ReviewNB, but I think the main suggestion is to rewrite the intro (now that you have the rest of it done) to let the reader know what the science question is and what tools we'll use to answer the question.</p>

**Comment by cpsanchez** [comment]
Thank you so much @jrka for all your comments, I believe I had taken all of them into consideration. Nevertheless I will come back to this tutorial in a couple of days to give it a second overall review and make sure all the recommendations have been implemented!
